src/screens/Coolers/containers/CoolerCardContainer.ts

```diff
@@ -3,8 +3,9 @@ import {connect} from 'react-redux';
 import CoolerCard from '../components/CoolerCard/CoolerCard';
 
 import {getCoolerByID} from '../selectors/coolersSelectors';
+import {NavigationComponent} from 'react-navigation';
 
-const mapStateToProps = (state: any, {navigation}: any) => {
+const mapStateToProps = (state: any, {navigation}: NavigationComponent) => {
```

Comment:
I think it will be useful to have index.d.ts file with types for every screen
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html